Tests/FoundationInternationalizationTests/GegorianCalendarInternationalizationTests.swift

```diff
@@ -23,7 +23,7 @@ import Testing
 @Suite("Gregorian Calendar (Internationalization)")
 private struct GregorianCalendarInternationalizationTests {
     @Test func copy() {
-        let gregorianCalendar = _CalendarGregorian(identifier: .gregorian, timeZone: nil, locale: nil, firstWeekday: 5, minimumDaysInFirstWeek: 3, gregorianStartDate: nil)
+        let gregorianCalendar = _CalendarGregorian(identifier: .gregorian, timeZone: .gmt, locale: nil, firstWeekday: 5, minimumDaysInFirstWeek: 3, gregorianStartDate: nil)
```

Comment:
This is a note to myself, but we should probably lift this logic of falling back to the current time zone out of `_CalendarGregorian` :/ Seems like a footgun even just for unit tests